pyblish_nukestudio/plugins/collect_selection.py

```diff
@@ -1,15 +1,13 @@
 import pyblish.api
 
-import hiero
 
-
-class CollectSelection(pyblish.api.Contextplugin):
+class CollectSelection(pyblish.api.ContextPlugin):
     """Inject the active project into context"""
 
-    version = (0, 1, 0)
     order = pyblish.api.CollectorOrder - 0.1
 
     def process(self, context):
+        import hiero
 
-        if hasattr(hiero, 'selection'):
-            context.set_data('selection', value=hiero.selection)
+        if hasattr(hiero, "selection"):
+            context.data["selection"] = hiero.selection
```

Comment:
I'm not familiar with Nuke Studio, but should publishing really be based on the active selection? I figure it'd make it tricky to publish non-interactively, if that's even possible.